### DISCUSSION TITLE
E-Mail

### DISCUSSION THREAD

**Discussion by kabelhai** [grindsa/acme2certifier#105] [Ideas]
Can the docker container send emails when for example a certificate expires?

**Comment by grindsa** [comment]
Just for better understanding: Which certificates do you want to monitor for expiry? The ones proxied by a2c or the certificate securing the local webserver?

**Comment by kabelhai** [comment]
The ones proxied by a2c.

**Comment by grindsa** [comment]
Hi, this is currently not possible. You could use the [a2c reporting feature](https://github.com/grindsa/acme2certifier/blob/master/docs/housekeeping.md) to get a certificate report and filter the expired certificates by yourself.  I am currently workig on a acme2certifier CLI which could also provide a list of expired certificates in either json or csv format. Would this be sufficient?

**Comment by kabelhai** [comment]
Yes, this would be sufficient. Thanks for your response.
